service/raft/raft_group0.cc

```diff
@@ -370,6 +370,7 @@ future<> raft_group0::abort() {
         return uninit_rpc_verbs(_ms.local());
     });
     co_await _shutdown_gate.close();
+    _leadership_monitor_as.request_abort();
```

Comment:
Shouldn't  we wait for `_leadership_monitor` future here?

Reply:
Yes, we should. I fixed it.